checkov/main.py

```diff
@@ -337,6 +338,13 @@ def add_parser_args(parser: ArgumentParser) -> None:
                help='Name for output file. The first selected output via output flag will be saved to the file (default output is cli)')
     parser.add('--output-bc-ids', action='store_true',
                help='Print Bridgecrew platform IDs (BC...) instead of Checkov IDs (CKV...), if the check exists in the platform')
+    parser.add('--include-all-checkov-policies', action='store_true',
+               help='When running with an API key, the Checkov will omit any policies that do not exist '
```

Comment:
```suggestion
               help='When running with an API key, Checkov will omit any policies that do not exist '
```
can be wrong, but sounds better without it 😄

Reply:
You are absolutely not wrong, lol